src/Microsoft.Extensions.Primitives/StringSegment.cs

```diff
@@ -35,7 +35,6 @@ public StringSegment(string buffer)
         /// <param name="buffer">The original <see cref="string"/> used as buffer.</param>
         /// <param name="offset">The offset of the segment within the <paramref name="buffer"/>.</param>
         /// <param name="length">The length of the segment.</param>
-        [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public StringSegment(string buffer, int offset, int length)
```

Comment:
This ctor will inline even without `AggressiveInlining`, so it was removed.

Reply:
This was on the edge in some MVC methods hence the AggressiveInlining

Reply:
OK. I'll reapply the attribute.

Bad idea to remove it, when it inlines anyway...